src/faq.html

```diff
@@ -4,6 +4,11 @@
 sections:
   - name: 
     faqs:
+      - title: >
+          What is the difference between Home Assistant SkyConnect and Connect ZBT-1?
+        content: >
+          <p>
+            Home Assistant Connect ZBT-1 is the new name for the Home Assistant SkyConnect. Though it's a new name and look, they have the exact same hardware, capability, and support. We made this change to help make room for a new line of Connect products, specifically an upcoming Z-Wave Connect product. Going forward, whenever we mention updates or features for the Home Assistant Connect ZBT-1, this will also apply to the SkyConnect-branded devices. Retailers will continue to sell SkyConnect-branded devices while supplies last, and we would encourage you to purchase whatever is available as they are functionally identical.</p>
```

Comment:
```suggestion
            Home Assistant Connect ZBT-1 is the new name for Home Assistant SkyConnect. Though it's a new name and look, they have the exact same hardware, capability, and support. We made this change to help make room for a new line of Connect products, specifically an upcoming Z-Wave Connect product. Going forward, whenever we mention updates or features for Home Assistant Connect ZBT-1, this will also apply to SkyConnect-branded devices. Retailers will continue to sell SkyConnect-branded devices while supplies last, and we would encourage you to purchase whatever is available as they are functionally identical.</p>
```